src/DataSkimmer.jl

```diff
@@ -104,11 +104,13 @@ end
 
 Skim any Tables.jl compatible table.
 """
-function skim(data)::Skimmed
-    if !Tables.istable(data)
+function skim(input_data)::Skimmed
+    if !Tables.istable(input_data)
```

Comment:
You could just remove this check, since it excludes, for example, a generator of NamedTuples, which can be a valid table, but isn't `Tables.istable`. If the input really isn't a table, it will throw an error in `Tables.columns` anyway.